actioncable/README.md

```diff
@@ -526,6 +526,18 @@ cable.subscriptions.create 'AppearanceChannel',
     # normal channel code goes here...
 ```
 
+## Download and Installation
+
+The latest version of Action Cable can be installed with RubyGems:
+
+```
+   $ gem install actioncable
+```
+
+Source code can be downloaded as part of the Rails project on GitHub
+
+* https://github.com/rails/rails/tree/master/actioncable
```

Comment:
Since we have the "npm and gem usage" part, this can be part of gem usage instead. Just the part about how to install. 
This README is non standard, it shows up on npm as well. So including this as is doesn't make sense.